index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="index.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link href="https://necolas.github.io/normalize.css/5.0.0/normalize.css" rel="stylesheet">
+  <meta charset="utf-8">
+</head>
+<body>
+  <div class="dock">
+    <h1 class="calculator-title">CALCULATOR</h1>
+    <div class="calculator-body" tabindex="0">
+      <div type="text" class="calculator-valueDisplay" maxlength="13"></div>
+      <div id="AC" class="calculator-button-clear">AC</div>
```

Comment:
try using a `data-something` attribute. `data-role` or `data-name` or whatever. just not `id` :P